src/javascript/app/pages/user/account/settings/self_exclusion.js

```diff
@@ -180,7 +178,7 @@ const SelfExclusion = (() => {
             validations.push({
                 selector        : `#${id}`,
                 validations     : checks,
-                exclude_if_empty: 1,
+                exclude_if_empty: 0,
```

Comment:
can we just remove the `exclude_if_empty` param instead of setting it to false? I believe by default it's false

Reply:
Ok, it's removed